content/de/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Über cgroup v2
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Auf Linux beschränken {{< glossary_tooltip text="control groups" term_id="cgroup" >}} die Ressourcen, die einem Prozess zugeteilt werden.
+
+Das {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} und die zugrundeliegende Container Runtime müssen mit cgroups interagieren um
+[Ressourcen-Verwaltung für Pods und Container](/docs/concepts/configuration/manage-resources-containers/) durchzusetzen. Das schließt CPU/Speicher Anfragen und Limits für containerisierte Arbeitslasten ein.
+
+Es gibt zwei Versionen cgroups in Linux: cgroup v1 und cgroup v2. cgroup v2 ist die neue Generation der `cgroup` API.
+
+<!-- body -->
+
+
+## Was ist cgroup v2? {#cgroup-v2}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+cgroup v2 ist die nächste Version der Linux `cgroup` API. cgroup v2 stellt ein einheitliches Kontrollsystem mit erweiterten Ressourcenmanagement Fähigkeiten bereit.
+
+cgroup v2 bietet einige Verbesserungen gegenüber cgroup v1, zum Beispiel folgende:
+
+- einzelnes vereinheitlichtes Hierarchiendesign in der API
```

Comment:
Shall these all start with an uppercase letter?
```suggestion
- Einzelnes vereinheitlichtes Hierarchiendesign in der API
```

Reply:
Yes, I think that would be more consistent. I'll implement this suggestion.